conf/nginx.conf

```diff
@@ -8,6 +8,12 @@ events {
 }
 
 http {
+  # Forward user's real ip in Cloudflare header.
+  # Avoid multiple proxy layers by using CF-Connecting-IP
+  # instead of X-Forwarded-For. 
+  set_real_ip_from 0.0.0.0/0;
```

Comment:
This looks fine, given that we're concerned about someone accidentally causing us problems rather than someone determined to cause an outage by submitting a large volume of requests. If we did have that problem we'd probably want to enforce that only traffic coming from Cloudflare is accepted in the first place.

As a side note, testing locally I find that the initial IP of the request is always one of the h Docker network. In AWS it seems to be the Docker bridge network. Both of these have a 172.0.0.0/8 IP, but the second octet is different. I guess there are other Docker networking configurations which would produce different IPs.